woocommerce/woocommerce-android
# [Shipping Labels Revamp] Finish Purchased screen UI feedback and data

Summary
==========
Fix issue #12288 by adding the following behaviors to the Purchased Shipping Label screen:
-  Shipping Label data transfer between the main Shipping Label form and the Purchased screen
- Wire the `WooShippingLabelPurchasedScreen` implementation with the respective ViewModel and ViewState
- Proper handling of Tracking and Pick Up options and their respective URLs
- Wire the original Shipping Labels `learn more` screen with the Label printing tooltip
- Loading and Error feedback
- `WooShippingLabelPurchasedViewModel` unit test coverage

Screen Capture
==========
https://github.com/user-attachments/assets/e0332965-6d33-443b-85f7-3c04a6db11e8

How to Test
==========
1. Open the main Shipping Label screen
2. Since the Purchased screen is only accessible from the landscape screen binding, switch the device to landscape mode
3. Hit the Purchase button
4. Once in the Purchased Screen, verify that the Shipping data is correctly presented, the `track shipment` and the `schedule pickup` buttons are working properly.
5. Hit the Print Shipping Label and verify that for each of the Paper size, the Shipping Label is opened in a separate screen following the selected paper size.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
